applications/deployments/osd-config-custom-domains-applications.adoc

```diff
@@ -1,9 +1,11 @@
 :_content-type: ASSEMBLY
 [id="osd-config-custom-domains-applications"]
-= Configuring custom domains for applications
+= Custom domains for applications
```

Comment:
According to the modular documentation guidelines - If the assembly describes a task and includes one or more procedure modules, start the title with a verb in the gerund form. 

Same applies to the procedure title

Reply:
I reverted the module to a gerund, but I do not think the assembly should have one since it's duplicating the headers. It currently looks like:

![image](https://user-images.githubusercontent.com/16167833/219493212-01ba0d09-22f1-4528-ab46-a0b1ae3c4fb5.png)

I changed it to:

![image](https://user-images.githubusercontent.com/16167833/219493122-d9bb1064-0f7c-499c-a312-71a19608ce31.png)